File: benchmarks/java/integer_boolean/Java_Primitive_Integer/Lift.java
import de.hhu.stups.btypes.BInteger;
import de.hhu.stups.btypes.BUtils;

public class Lift {






    private BInteger floor;

    public Lift() {
        floor = (BInteger) new BInteger(0);
    }

    public void inc() {
        floor = (BInteger) floor.plus(new BInteger(1));
    }

    public void dec() {
        floor = (BInteger) floor.minus(new BInteger(1));
    }



}

File: benchmarks/C++_Primitives/Lift.cpp
#include <iostream>
#include <string>
#include "BUtils.cpp"
#include "BInteger.cpp"

#ifndef Lift_H
#define Lift_H

using namespace std;

class Lift {

    public:

    private:




        BInteger floor;

    public:

        Lift() {
            floor = (BInteger(0));
        }

        void inc() {
            floor = floor.plus((BInteger(1)));
        }

        void dec() {
            floor = floor.minus((BInteger(1)));
        }

};


#endif